2_gyum/aladin.py

```
@@ -0,0 +1,168 @@
+
+import pandas as pd
+
+
+# In[46]:
+
+
+datas = []
+for page_num in range(1, 4):
+     
+    url = f"https://www.aladin.co.kr/shop/common/wbest.aspx?BestType=Bestseller&BranchType=1&CID=0&page={page_num}&cnt=1000&SortOrder=1"
+    response = requests.get(url)  # 요청 보내기
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)